### PR TITLE
layout and login has been finished

### PR DESCRIPTION
**Pull request by Oxunjon1029**

### CODE REVIEW

File: client/budgedtify/src/app/account/account-front/account-front.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'app-account-front',
+  templateUrl: './account-front.component.html',
+  styleUrls: ['./account-front.component.css']
+})
+export class AccountFrontComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (AudriusBaltusis) [inline client/budgedtify/src/app/account/account-front/account-front.component.ts]: Please remove empty `ngOnInit`

File: client/budgedtify/src/app/app.component.ts [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
   title = 'budgedtify';
+
+  constructor(private authservice: AuthService, private router: Router) {}
+
+  get isLoggedIn() {
```
Comment (AudriusBaltusis) [inline client/budgedtify/src/app/app.component.ts]: Please add return type

File: client/budgedtify/src/app/auth/auth-form/auth-form.component.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
+* {
```
Comment (AudriusBaltusis) [inline client/budgedtify/src/app/auth/auth-form/auth-form.component.css]: Please use `sass`/`scss`

File: client/budgedtify/src/app/auth/auth-form/auth-form.component.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+              *ngIf="isValidEmail['email'].errors?.['required'] && isValidEmail['email'].dirty"
+            >
+              Required field is empty
+            </mat-error>
+            <mat-error
+              *ngIf="isValidEmail['email'].errors?.['email'] && isValidEmail['email'].dirty"
+            >
+              Please enter valid email
+            </mat-error>
+            <!-- <mat-error *ngIf="isValid('email')"
```
Comment (AudriusBaltusis) [inline client/budgedtify/src/app/auth/auth-form/auth-form.component.html]: Please remove unused code

File: client/budgedtify/src/app/auth/auth-form/auth-form.component.ts [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    email: new FormControl('', [
+      Validators.required,
+      Validators.email,
+    ]),
+    password: new FormControl('', Validators.required),
+  });
+
+  responsedata: any;
+  submitted = false;
+  // isValid(controlname: string) {
```
Comment (AudriusBaltusis) [inline client/budgedtify/src/app/auth/auth-form/auth-form.component.ts]: Unused code

File: client/budgedtify/src/app/auth/auth-form/auth-form.component.ts [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  //   return (
+  //     this.loginForm.get(controlname)?.invalid &&
+  //     this.loginForm.get(controlname)?.touched
+  //   );
+  // }
+
+  get isValidEmail() {
+    return this.loginForm.controls;
+  }
+  onSubmit() {
```
Comment (AudriusBaltusis) [inline client/budgedtify/src/app/auth/auth-form/auth-form.component.ts:28]: return type

**Review by AudriusBaltusis — CHANGES_REQUESTED** [review]

**Comment by Oxunjon1029** [comment]
I fixed the problems. Could you check them out?

**Comment by AudriusBaltusis** [comment]
Some return types are still missing. Please fix this in next PR

**Review by AudriusBaltusis — APPROVED** [review]

**Comment by Oxunjon1029** [comment]
Ok , I will fix them in the next PR

**Pull request merged**